src/test/journal/test_ObjectRecorder.cc

```diff
@@ -70,6 +70,10 @@ class TestObjectRecorder : public RadosTestFixture {
 	C_SaferCond cond;
 	object_recorder->flush(&cond);
 	cond.wait();
+	std::scoped_lock l{*m_handler.object_lock};
+	if (!object_recorder) {
```

Comment:
Should this really be testing that the pointer is null?

Reply:
no. and sorry, i pushed a wrong branch.

Reply:
fixed and repushed,